MachObfuscator/main.swift

```diff
@@ -4,10 +4,11 @@ private func main() {
     let options = Options.fromCommandLine()
     guard !options.help,
         let appDirectory = options.appDirectory,
-        let mangler = SymbolManglers.mangler(byKey: options.manglerKey) else {
+        let manglerType = SymbolManglers(rawValue: options.manglerKey) else {
```

Comment:
You leveraged enums, nice :) What do you think about parsing `manglerKey` in `Options`? E.g. `appDirectory` is parsed from `String` to `URL` in `Options`. So the `Options.manglerKey` would change to `Options.manglerType`, and this line would look like:
```suggestion
        let manglerType = options.manglerType else {
```